.gitignore

```diff
@@ -1,4 +1 @@
-.project
-.env
-.vscode/*
-backup/config.cfg
\ No newline at end of file
+config.cfg
```

Comment:
As so many folk use vscode, useful to have those ignores in.  I'd moved the config file so it could always be found :)